data-raw/otu_mini.R

```diff
@@ -5,56 +5,41 @@ usethis::use_data(otu_mini, overwrite = TRUE)
 
 test_hyperparams <- structure(list(
   param = c(
-    "cost", "cost", "cost", "loss", "epsilon",
-    "sigma", "sigma", "C", "C", "maxdepth", "maxdepth", "nrounds",
-    "gamma", "eta", "max_depth", "colsample_bytree", "min_child_weight",
-    "subsample", "mtry", "mtry"
+    "lambda", "lambda", "lambda", "alpha",
+    "sigma", "sigma", "C", "C",
+    "maxdepth", "maxdepth",
+    "nrounds", "gamma", "eta", "max_depth", "colsample_bytree", "min_child_weight", "subsample",
+    "mtry", "mtry"
   ),
   value = c(
-    "1e-3", "1e-2", "1e-1", "L2_primal", "0.01", "0.00000001", "0.0000001", "0.01", "0.1",
-    "1", "2", "10", "0", "0.01", "1", "0.8", "1", "0.4", "1", "2"
+    "1e-3", "1e-2", "1e-1", "1",
+    "0.00000001", "0.0000001", "0.01", "0.1",
+    "1", "2",
+    "10", "0", "0.01", "1", "0.8", "1", "0.4",
+    "1", "2"
   ),
   method = c(
-    "regLogistic", "regLogistic", "regLogistic", "regLogistic",
-    "regLogistic", "svmRadial", "svmRadial", "svmRadial", "svmRadial",
-    "rpart2", "rpart2", "xgbTree", "xgbTree", "xgbTree", "xgbTree",
-    "xgbTree", "xgbTree", "xgbTree", "rf", "rf"
+    "glmnet", "glmnet", "glmnet", "glmnet",
+    "svmRadial", "svmRadial", "svmRadial", "svmRadial",
+    "rpart2", "rpart2",
+    "xgbTree", "xgbTree", "xgbTree", "xgbTree", "xgbTree", "xgbTree", "xgbTree",
+    "rf", "rf"
   )
 ),
-class = c("spec_tbl_df", "tbl_df", "tbl", "data.frame"), row.names = c(NA, -20L)
+class = c("spec_tbl_df", "tbl_df", "tbl", "data.frame"), row.names = c(NA, -19L)
 )
 
 ## code to prepare `otu_mini_results`
 # includes grouping functionality & feature importance
 set.seed(2019)
-# otu_mini_group <- sample(LETTERS[1:10], nrow(otu_mini), replace = TRUE)
-otu_mini_group <- c(
-  "I", "J", "E", "A", "H", "G", "C", "J", "I", "J", "A", "H",
-  "C", "G", "H", "C", "D", "E", "C", "D", "E", "G", "I", "A", "G",
-  "F", "F", "A", "J", "G", "F", "E", "A", "F", "E", "J", "F", "A",
-  "B", "A", "A", "I", "I", "C", "A", "H", "J", "G", "G", "B", "F",
-  "F", "I", "J", "H", "G", "F", "H", "H", "C", "I", "E", "B", "B",
-  "I", "H", "G", "C", "G", "G", "I", "F", "I", "D", "J", "H", "C",
-  "F", "C", "F", "E", "C", "B", "B", "D", "G", "F", "F", "J", "B",
-  "B", "G", "G", "J", "B", "J", "J", "G", "D", "G", "H", "I", "H",
-  "D", "G", "I", "F", "A", "E", "C", "B", "B", "E", "J", "F", "H",
-  "C", "F", "C", "D", "I", "H", "A", "G", "E", "F", "A", "C", "E",
-  "I", "D", "A", "C", "D", "H", "A", "A", "J", "F", "E", "C", "J",
-  "J", "G", "C", "D", "H", "E", "E", "F", "G", "F", "C", "E", "F",
-  "D", "D", "B", "J", "B", "H", "A", "A", "A", "B", "D", "J", "D",
-  "F", "F", "B", "G", "J", "B", "F", "G", "F", "J", "B", "D", "B",
-  "C", "C", "H", "B", "F", "I", "G", "I", "D", "G", "G", "E", "F",
-  "I", "B", "B", "I", "J", "A"
-)
-otu_mini_results1 <- mikropml::run_ml(otu_mini, # use built-in hyperparams
-  "regLogistic",
+otu_mini_group <- sample(LETTERS[1:5], nrow(otu_small), replace = TRUE)
```

Comment:
I was getting inconsistent results with `sample()` even with `set.seed()` immediately before it. So I pasted its `dput()` results instead of having data-raw run it again every time.

Reply:
Done. Letters are different, but now hard-coded.